src/cljs/main/broadfcui/common/markdown.cljs

```diff
@@ -32,29 +32,37 @@
 
 (react/defc MarkdownEditor
   {:get-trimmed-text
-   (fn [{:keys [state]}]
+   (fn [{:keys [state this]}]
+     (assert (not (this :-managed?)) "Trying to read text from a managed component")
```

Comment:
I'm confused by what `:-managed?` does. I thought maybe it was a React thing, but I can't find anything in the docs

Reply:
Hmm, proper language appears to be "controlled", though I swear I've heard "managed". Anyway: https://reactjs.org/docs/uncontrolled-components.html